OpenSim/Simulation/Model/Probe.h

```diff
@@ -123,8 +123,8 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Probe, ModelComponent);
 //==============================================================================
 // PROPERTIES
 //==============================================================================
-    /** Enabled by default. **/
-    OpenSim_DECLARE_PROPERTY(isDisabled, bool,
+    /** Enabled (true) by default. **/
+    OpenSim_DECLARE_PROPERTY(enabled, bool,
```

Comment:
Comment on L128 should be revised to "Flag indicating whether the Probe is enabled"